pyaedt/hfss.py

```diff
@@ -2373,12 +2373,12 @@ def edit_source(self, portandmode, powerin, phase="0deg"):
         >>> wave_port = hfss.create_wave_port_from_sheet(sheet, 5, hfss.AxisDir.XNeg, 40,
         ...                                              2, "SheetWavePort", True)
         >>> hfss.edit_source("SheetWavePort" + ":1", "10W")
-        pyaedt Info: Setting up power to Eigenmode 10WWatt
+        pyaedt Info: Setting up power to Eigenmode 10W
         True
 
         """
 
-        self._messenger.add_info_message("Setting up power to Eigenmode " + powerin + "Watt")
+        self._messenger.add_info_message("Setting up power to Eigenmode " + powerin)
```

Comment:
```suggestion
        self._messenger.add_info_message("Setting up power to Eigenmode " + ``powerin``.)
```

Reply:
@MaxJPRey I'm not sure if the literal tagging is acceptable within a message, but if so, it seems more readable, especially when concluding the sentence with a period.

Reply:
as part of a string passed to an argument we should not use ``. Original command from @MaxJPRey was correct